labgrid/driver/qemudriver.py

```diff
@@ -183,6 +183,9 @@ def on_activate(self):
     def on_deactivate(self):
         if self.status:
             self.off()
+        if self._clientsocket:
+            self._clientsocket.close()
+        self._socket.close()
```

Comment:
We should set them to `None` after closing, to match the initial state.

Reply:
Agreed.

Reply:
Done.